pkg/apis/extensions/types.go

```diff
@@ -88,6 +88,14 @@ type ResourceConsumption struct {
 	Quantity resource.Quantity `json:"quantity,omitempty"`
 }
 
+type Utilization float64
+
+type CPUTargetUtilization struct {
+	// Utilization target defines fraction of the requested CPU that should be utilized/used,
+	// e.g. 0.7 means that 70% of the requested CPU should be in use.
+	UtilizationTarget Utilization `json:"utilizationTarget"`
```

Comment:
I understand the desire to make this similar to the GCE autoscaling API, but:

1. "Utilization" is redundant, since it's in the parent field's name.
2. Floats are evil and unnecessary. 

How about TargetPercentage int?

https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api-conventions.md#naming-conventions